eq-author/src/App/QuestionnairesPage/QuestionnairesTable/Row.test.js

```diff
@@ -24,29 +38,16 @@ describe("Row", () => {
         name: "Alan",
       },
     };
-    handleDeleteQuestionnaire = jest.fn();
-    handleDuplicateQuestionnaire = jest.fn();
   });
 
   it("should render", () => {
-    const wrapper = shallow(
-      <Row
-        questionnaire={questionnaire}
-        onDuplicateQuestionnaire={handleDuplicateQuestionnaire}
-        onDeleteQuestionnaire={handleDeleteQuestionnaire}
-      />
-    );
+    const wrapper = shallow(<Row questionnaire={questionnaire} {...props} />);
```

Comment:
Could we move questionnaire into props, so it just becomes `<Row {...props}>`?

Reply:
Done.